internal/handshake/unsafe.go

```diff
@@ -19,6 +19,9 @@ func init() {
 	if !structsEqual(&tls.ClientSessionState{}, &qtls.ClientSessionState{}) {
 		panic("qtls.ClientSessionState not compatible with tls.ClientSessionState")
 	}
+	if !structsEqual(&tls.ClientSessionState{}, &clientSessionState{}) {
+		panic("clientSessionState not compatible with tls.ClientSessionState")
```

Comment:
Should we add a note here to file a bug if this ever happens?

Reply:
I think we'll always be aware of this, since our CI will catch it.